### DISCUSSION TITLE
Lack of performance on virtualised table on Windows and Linux

### DISCUSSION THREAD

**Discussion by JdeJ** [komarovalexander/ka-table#176] [Q&A]
Hi Alexander, I have a table virtualised with a lot of data with custom content cells, I develop in Mac and it works fine, but in Windows works very bad (with native cells also occurs). If I disable virtualisation all works well.

Also there was a bug on virtualised tables that trigger infinite scrolling but it has been solved by last version (7.0.0).

You can test it here:
https://stackblitz.com/edit/table-many-columns-ts-xkx6fu?file=Demo.tsx

Thanks

**Comment by komarovalexander** [comment]
Hi @JdeJ ,
 Try [memorization](https://komarovalexander.github.io/ka-table/#/many-rows-memo) it can help you to solve that issue
https://stackblitz.com/edit/table-many-columns-ts-puk958?file=Demo.tsx
Let me know please if it does not work for you
